kpatch-build/kpatch-build

```diff
@@ -267,7 +267,7 @@ find_core_symvers() {
 }
 
 gcc_version_from_file() {
-	readelf -p .comment "$1" | grep -m 1 -o 'GCC:.*'
+	"$CROSS_COMPILE"readelf -p .comment "$1" | grep -m 1 -o 'GCC:.*'
 }
```

Comment:
Also, instead of sprinkling CROSS_COMPILE everywhere, it may be cleaner to just have GCC, CLANG, READELF, OBJCOPY, LD, LD_LLD variables defined at the top of the file to include CROSS_COMPILE.

(As a further eventual cleanup we might want to just have CC and LD variables instead of GCC/CLANG/LD/LD_LLD, but that might be a little tricky since it depends on the config file, and maybe can be done later.)

Reply:
I do not know if these changes work with clang as well, so I can add them at our own risk